### PR TITLE
Computed props nano

### PR DESCRIPTION
**Pull request by marina-mosti**

### CODE REVIEW

**Review by jlooper — COMMENTED** [review]
This looks terrific. I really like the simplicity although the concept is hard to grasp. Having the explanation in a tip at the end is probably good, although I suppose we could add it mid-stream.

Barring comments from @NataliaTepluhina I'd like to merge / copyedit etc.

**Review by NataliaTepluhina — CHANGES_REQUESTED** [review]

**Comment by NataliaTepluhina** [comment]
@immarina thank you for your hard work! I really love the clear explanations you give on computed properties and how they work, In the meantime, I would probably ask you to replace an example: with current one we don't have any reactive dependencies. So, in fact, this computed property will _never_ update and we can't show one of the core features of computed: recalculating on dependency change.

File: workshop/nanos/nano4.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+Right now it won't work, because `today` is a property that hasn't been defined yet. You will do that shortly.
+
+## Creating a computed property
+
+Computed properties are `functions` that can calculate a value and return data that will be used in our template. In our case, we will use them to create a `Date` object, and return it's string representation only.
+
+This type of logic would be cumbersome to maintain if placed inside line blocks on our template, and may even be impossible if the logic is too complex due to the templating engine limitations.
+
+Rule of thumb is, whenever you need to `calculate` a value, or change a value in any way - use computed properties.
```
[the remark this answers is not in the document]
Comment (marina-mosti) [reply workshop/nanos/nano4.md]: Will rephrase to
`Rule of thumb is, whenever you need to `calculate` a value reactively when another property changes - use computed properties.`

**Review by jlooper — COMMENTED** [review]
This looks better to me, to show an actual calculation against a date, and something a student can watch working in real time. I leave it to @NataliaTepluhina for final approvals

**Comment by NataliaTepluhina** [comment]
Thank you for changing it, @immarina ! Looks good to me now 👍

**Review by NataliaTepluhina — APPROVED** [review]

**Pull request merged**